### ISSUE TITLE
TwoFactor Gateway not option in 'Enforce Two Factor'

### ISSUE THREAD

**Issue by anatomism** [nextcloud/twofactor_gateway#289]
With recently released NC17, one feature I like and use is the Enforce 2FA - the new feature is whereby a new user logs in for the first time and they get the option to setup 2FA (rather than just locking them out as it used to).

The issue is, the TwoFactor Gateway isn't one of the options - I have the gateway configured with SMS (all working), as well as TOTP.  Only the TOTP option is given for a new user. 

I'd like for the SMS to become primary eventually, but cannot if a new user cannot set it up

Thanks

**Comment by ChristophWurst** [comment]
Yeah, this should be possible. It just needs someone to implement it :)

Ref https://github.com/nextcloud/twofactor_u2f/pull/507 https://github.com/nextcloud/twofactor_totp/pull/550 for the other implementations.
